src/pages/portfolio/Overview.tsx

```diff
@@ -42,10 +54,16 @@ export const Overview = () => {
                   PositionsTableColumnKey.UnrealizedPnl,
                   PositionsTableColumnKey.RealizedPnl,
                   PositionsTableColumnKey.AverageOpenAndClose,
-                ]
+                  shouldRenderTriggers && PositionsTableColumnKey.Triggers,
+                  shouldRenderActions && PositionsTableColumnKey.Actions,
+                ].filter(isTruthy)
           }
           currentRoute={AppRoute.Portfolio}
-          withGradientCardRows
```

Comment:
re this table -- did achal mention if we should add back borders/ border radius so they look the same as other tables in Portfolio?

Reply:
I also recommend re-adding the borders/border-radius :)

Reply:
<img width="1359" alt="Screenshot 2024-03-06 at 6 37 47 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/4a410af9-cb7a-4937-ae10-b7b4ddbfe0cf">

Reply:
<img width="1359" alt="Screenshot 2024-03-06 at 6 37 47 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/7540b419-0321-4b14-81bd-7f10b1115f3f">